### PR TITLE
[desktop] Help Menu links maintenance (#633)

### PR DESCRIPTION
**Pull request by votaguz**
1) Removed:
    - Order Frostwire Stickers
    - Frostwire T-Shirts
2) Added : 
    - Support Frostwire links to http://www.frostwire.com/give
3) Updated:
    - Frostwire Forums links to http://forum.frostwire.com/

### CODE REVIEW

**Comment by aldenml** [comment]
LGTM, but since this more about links, wording and web, @gubatron needs to approve it.

File: desktop/src/com/limegroup/gnutella/gui/menu/HelpMenu.java [hunk 1]
```diff
@@ -54,9 +54,7 @@
 
         addSeparator();
 
-        addMenuItem(new OpenLinkAction("http://www.frostwire.com/stickers",I18n.tr("Order FrostWire Stickers"),I18n.tr("Order FrostWire stickers, we ship worldwide.")));
-
-        addMenuItem(new OpenLinkAction("http://www.frostwire.com/shop",I18n.tr("FrostWire T-Shirts"),I18n.tr("Get a T-shirt, support the project. You'll get also 10 free stickers")));
+        addMenuItem(new OpenLinkAction("http://www.frostwire.com/give",I18n.tr("Support Frostwire"),I18n.tr("Support Frostwire")));
```
Comment (gubatron) [inline desktop/src/com/limegroup/gnutella/gui/menu/HelpMenu.java]: append this to the URL: `?from=desktop-help` ->
`http://www.frostwire.com/give?from=desktop-help`

File: desktop/src/com/limegroup/gnutella/gui/menu/HelpMenu.java [hunk 2]
```diff
@@ -67,7 +65,7 @@
 
         addMenuItem(new OpenLinkAction("http://twitter.com/#!/frostwire", I18n.tr("Follow us on Twitter"), I18n.tr("Follow us on Twitter")));
 
-        addMenuItem(new OpenLinkAction("http://www.frostwire.com/forum/viewforum.php?f=1", I18n.tr("Foru&m"), I18n.tr("Access the FrostWire Users\' Forum")));
+        addMenuItem(new OpenLinkAction("http://forum.frostwire.com/", I18n.tr("Foru&m"), I18n.tr("Access the FrostWire Users\' Forum")));
```
Comment (gubatron) [inline desktop/src/com/limegroup/gnutella/gui/menu/HelpMenu.java]: Please append this to the URL: `?from=desktop-help` -> `http://forum.frostwire.com/?from=desktop-help`

File: desktop/src/com/limegroup/gnutella/gui/menu/HelpMenu.java (hunk 1 above)
Comment (gubatron) [reply desktop/src/com/limegroup/gnutella/gui/menu/HelpMenu.java]: Frostwire has been misspelled, it's `FrostWire` (capital W)

**Comment by gubatron** [comment]
Also, there's more to be done:

- FAQ Link is broken.
Should point to -> `https://support.frostwire.com/hc/en-us/categories/200015110-FAQs`
- Rename "FrostWire on Facebook" to "Follow us on Facebook"
- Add a link to [Follow us on Instagram](https://instagram.com/frostwire)
- Add a link to [Follow us on G+](https://plus.google.com/+frostwire/posts)

**Comment by davisjackie321** [comment]
Well

**Pull request merged**